docs/source/index.rst

```diff
@@ -0,0 +1,30 @@
+Welcome to CSD3's documentation
```

Comment:
There should be a hierarchy for each machine we have i.e. one for FREIA, one for Cumulus, one for CSD3 and so on

Reply:
i.e. there should be another level of hierarchy